cmd/addons.go

```diff
@@ -160,4 +160,46 @@ var (
 			autocomplete.CmdFlagsAutoComplete(c, "addons-info")
 		},
 	}
+	addonsConfigCommand = cli.Command{
+		Name:     "addons-config",
+		Category: "Addons",
+		Usage:    "Configure an add-on attached to your app",
+		Flags: []cli.Flag{&appFlag,
+			&cli.StringFlag{Name: "maintenance-window-hour", Usage: "Configure add-on maintenance window starting hour"},
+			&cli.StringFlag{Name: "maintenance-window-day", Usage: "Configure add-on maintenance window day"},
+		},
+		ArgsUsage: "addon-id",
+		Description: CommandDescription{
+			Description: "Configure an add-on attached to your app",
+			Examples:    []string{"scalingo --app manifest addons-config --maintenance-window-day tuesday --maintenance-window-hour 2 addon_uuid"},
```

Comment:
suggestion: It should not only take an addon UUID. It should be possible to provide the database type (e.g. `postgresql`). Cf. what is done in the `logs` command: https://github.com/Scalingo/cli/blob/6ef569ab159e38e69ef8372d6e287dd197796916/db/logs.go#L28-L35

consistency: example app name is _always_ `my-app`. Not `manifest`.

Reply:
Oops my bad here